services/cd-service/Dockerfile

```diff
@@ -11,6 +11,7 @@ ADD services/cd-service/pkg /kp/services/cd-service/pkg
 ADD pkg /kp/pkg
 COPY go.sum go.mod /kp/
 
+RUN make -C /kp/pkg/api
```

Comment:
if the cd-service needs it, I would assume the rollout-service needs it too. -> please add it there.

Reply:
Done